src/main/java/core/basesyntax/strategy/QuantityCounterStrategyImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.Main;
+import core.basesyntax.operation.Transaction;
+import core.basesyntax.service.QuantityCounter;
+import java.util.Map;
+
+public class QuantityCounterStrategyImpl implements QuantityCounterStrategy {
+    private Map<Transaction.Operation, QuantityCounter> quantityCounterMap = Main.get();

```

Comment:
get() naming is not so informative naming, better use getOperationMao or smth else

Reply:
and for strategy you can create own util class